backend/src/modules/idim-webservice/idim-webservice.service.ts

```diff
@@ -132,4 +142,109 @@ export class IdimWebserviceService {
             );
         }
     }
+
+    async verifyBceidUser(
+        userId: string,
+        requesterUserGuid: string,
+        requesterAccountTypeCode: string
+    ): Promise<HttpException | BCEIDUserResponse> {
+        if (!this.hasAllRequiredParams()) {
+            return new HttpException(
+                {
+                    error: 'Missing IDIM web service crednetials in the environment variables',
+                },
+                HttpStatus.INTERNAL_SERVER_ERROR
+            );
+        }
```

Comment:
Hi Catherine, for this service to work, this "if" logic is the prerequisite for the entire class, moving to "constructor" make sense more so there is no need for both IDIR/BCEID verifying methods to call the same.

Reply:
Thanks for verifying!